pkgs/applications/networking/protonvpn-linux-cli/default.nix

```diff
@@ -17,6 +17,7 @@ python3Packages.buildPythonApplication rec {
       setuptools
       jinja2
       pythondialog
+      jinja2
```

Comment:
This line should be removed as jinja2 is added twice.

Reply:
Removed duplicate `jinja2`